orttraining/orttraining/python/training/_ortmodule_io.py

```diff
@@ -298,13 +323,25 @@ def parse_inputs_for_onnx_export(all_input_parameters, onnx_graph, inputs, kwarg
                 if inp.requires_grad:
                     # input_names_require_grad holds all input tensors that have requires_grad
                     input_names_require_grad.append(name)
-
                 input_names.append(name)
                 dynamic_axes[name] = {}
                 for dim_idx in range(len(inp.shape)):
-                    dynamic_axes[name].update({dim_idx: f'input{input_idx}_dim{dim_idx}'})
-
+                    dynamic_axes[name].update({dim_idx: f'{name}_dim{dim_idx}'})
                 input_shape.append(list(inp.size()))
+        elif input_parameter.kind == inspect.Parameter.VAR_KEYWORD:
+            # **kwargs is always the last argument of forward()
+            for name,inp in kwargs.items():
+                if name not in input_names:
+                    print(f'{input_idx} VAR_KEYWORD {input_parameter.kind} including as keyword/kwargs {name}********************************************************************************************************************')
+                    if inp is not None and (onnx_graph is None or name in onnx_graph_input_names):
+                        if inp.requires_grad:
+                            # input_names_require_grad holds all input tensors that have requires_grad
+                            input_names_require_grad.append(name)
+                        input_names.append(name)
+                        dynamic_axes[name] = {}
+                        for dim_idx in range(len(inp.shape)):
+                            dynamic_axes[name].update({dim_idx: f'{name}_dim{dim_idx}'})
+                        input_shape.append(list(inp.size()))
```

Comment:
Is there a way to refactor this part of the code so that we only need to write it once for all parameter-kinds? Perhaps extract it to a local function?